addons/website_blog/static/src/js/website_blog.js

```diff
@@ -71,10 +53,11 @@ publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
     _onContentAnchorClick: function (ev) {
         ev.preventDefault();
         ev.stopImmediatePropagation();
-        var element = ev.currentTarget;
-        var target = $(element.hash);
+        var $target = $(ev.currentTarget.hash);
+        var self = this;
```

Comment:
Useless `self`

Reply:
Just a confirmation, it it ok to use `this` rather than `self` inside the `.animate()` function ([line 60](https://github.com/odoo/odoo/pull/34609/commits/11b43e4230c6b027c013d70fcae4b57e728679d3#diff-856438c11d77f1149ba3f0f41fc074f0R60))?
It seems working fine btw, just to be sure

Reply:
It is not in a new function, it is in the same context. You could write the line like this:
```javascript
var options = {scrollTop: this.blablabla};
$('html, body').stop().animate(options);
```

Note that we begin using ES6 now, so even for anonymous functions, you can use `this` if you use the arrowed syntax:
Before:
```javascript
var self = this;
myArray.forEach(function (value) {
    // You must use 'self' here
});
```
Now:
```javascript
myArray.forEach(value => {
    // You can use 'this' here
});
```